code/ui/blocks/src/blocks/Meta.mdx

```diff
@@ -0,0 +1,47 @@
+# The `<Meta />` Block
+
+The `<Meta />` block is a special block that is used to define the meta of the stories (the stories' default export) in the file.
+It's not necessary when writing docs within Storybook (it doesn't do anything in that case),
+but it _is_ required when writing Portable Docs.
+
+As an example, let's pretend we wanted to include stories for a `<Button />` component in our docs outside of Storybook.
+The MDX file would look like this:
+
+```md
+import { Meta, Story, Canvas } from '@storybook/blocks';
+import \* as ButtonStories from '../stories/Button.stories';
```

Comment:
Something annoying is going on here:

1. Our syntax highlighter doesn't support MDX, only MD, so I've used that here.
2. When using the `md` language, Prettier will automatically try to escape the `*`, which it shouldn't do here, because the backslash will actually show up in the docs.
3. Our MDX parser doesn't currently support comments in MDX2 like `{/* prettier-ignore */}` - it breaks with `Extraction error on ui/blocks/src/blocks/Meta.mdx: Error: Unexpected JSX child: JSXExpressionContainer`, so there's no way for me disable Prettier here. I don't know if this is already on your radar @shilman ?

When we've fixed 3, we should be able to revisit this and remove the backslash.